recipes/sphinxcontrib-jquery/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "sphinxcontrib-jquery" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "8fb65f6dba84bf7bcd1aea1f02ab3955ac34611d838bcc95d4983b805b234daa" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=2.7
+    - pip
+    - flit-core >=3.7,<4
+  run:
+    - python >=2.7
+    - setuptools
+    - sphinx
+
+test:
+  imports:
+    - sphinxcontrib.jquery
+  requires:
+    - pip
+    - pytest
+  source_files:
+    - tests
+  commands:
+    - pip check
+    - python -m pytest tests -vv
```

You could try with this, but first check that the PyPI tarball is actually packaging the tests.

```suggestion
  source_files:
    - tests/*
  commands:
    - pip check
    - ls -alh .
    - python -m pytest tests/ -vv
```

---

Just confirmed (by downloading the [latest tarball from PyPI](https://files.pythonhosted.org/packages/38/e8/8ba91a585a8d0b6a1952eeeb8ea392e025e4efc7c860f4679794a776b116/sphinxcontrib-jquery-2.0.0.tar.gz)) that tests are not being packaged.

Pinging @AA-Turner for confirmation.

---

You might want to use the github tarball in that case. However, you are not required to run the full test suite on conda-forge as part of the packaging submission! It's ok if you want to run it once or twice before leaving a simple test section to ensure things are working, though.

---

@jaimergp @bollwyvl Thanks for the insightful comments!

I've been thinking what would be the best way forward with this recipe, based on your suggestions.
I think that an upstream PR to add the `tests` into the sdist would be the most elegant way forward.
This way, whenever said PR is merged we can update this recipe to do the packaged tests.

Till that time however, since testing _is_ [being done upstream](https://github.com/sphinx-contrib/jquery/commit/08d948a4a22961c83f468fc5dd2200fc411c0c23), I feel that we can keep this recipe simple and sweet, doing only the (relatively) simple `import` and `pip check` tests.

@AA-Turner does that seem like a sensible course of action to you as well?